81-save-an-image-locally.md

```diff
@@ -0,0 +1,12 @@
+# How To Save An Image Locally Using Python Whose URL Address I Already Know?
```

Comment:
I do not think you need need to start every work upper case

Reply:
I didn't notice I only copy the question. I will modify it!